how-it-works/1_protocol/02_consensus.subpage.md

```diff
@@ -5,14 +5,118 @@ shareImage: /img/how-it-works/consensus.600.jpg
 slug: consensus
 ---
 
-# Internet Computer Consensus
-<!-- https://img.youtube.com/vi/vVLRRYh3JYo/0.jpg -->
-[![Watch youtube video](https://i.ytimg.com/vi/vVLRRYh3JYo/maxresdefault.jpg)](https://www.youtube.com/watch?v=vVLRRYh3JYo)
+# Consensus
+
+Each subnet of the IC is its own blockchain that makes progress concurrently to the other subnets and thus runs its own instance of the IC core protocol stack including consensus.
+Recall that the goal of consensus is to produce blocks agreed by the nodes of the subnet with an ordered set of messages to be executed.
+This is crucial so that the upper two layers of the protocol stack – message routing and execution – receive the same inputs in every round on each node.
+
+The IC's consensus protocol is a novel protocol that achieves a high throughput both in terms of messages and message payload as well as a short time to finality.
+Block finality on the IC is cryptographic and thus can have very short latency.
```

Comment:
Uhhhh...this is kinda repetitive, especially if you change the previous sentence as suggested.

Maybe these two sentences can become: 
The IC's consensus protocol is a novel protocol that achieves a high throughput and low latency.
This low latency is possible because block finality on the IC is cryptographic.